### ISSUE TITLE
[Feature Request] Support C# language

### ISSUE THREAD

**Issue by yiyione** [tjovicic/algorithm-contest-scrapper#2]
The default .vscode folder is for g++ by default.
Is it possible to support dotnet core?

Thanks!

**Comment by tjovicic** [comment]
Yes, we can. You are free to contribute or explain me how to do it. I don't know much about dotnet
